# HG changeset patch
# User Mike Miller <mtmiller@ieee.org>
# Date 1412650452 14400
#      Mon Oct 06 22:54:12 2014 -0400
# Node ID cfa9996afcbfdd7740dac8029d1dcc0f6fb8f1fe
# Parent  4122df73e374254c19271476e9cc465f2a7a9eb0
doc: fix stray escape sequence in mkstemp's docstring

* file-io.cc (Fmkstemp): Fix stray escape sequence in docstring introduced
in cset 841d8f606bcd.

diff --git a/libinterp/corefcn/file-io.cc b/libinterp/corefcn/file-io.cc
--- a/libinterp/corefcn/file-io.cc
+++ b/libinterp/corefcn/file-io.cc
@@ -2040,17 +2040,17 @@ DEFUN (mkstemp, args, ,
 @deftypefnx {Built-in Function} {[@var{fid}, @var{name}, @var{msg}] =} mkstemp (\"@var{template}\", @var{delete})\n\
 Return the file descriptor @var{fid} corresponding to a new temporary file\n\
 with a unique name created from @var{template}.\n\
 \n\
 The last six characters of @var{template} must be @qcode{\"XXXXXX\"} and\n\
 these are replaced with a string that makes the filename unique.  The file\n\
 is then created with mode read/write and permissions that are system\n\
 dependent (on GNU/Linux systems, the permissions will be 0600 for versions of\n\
-glibc\ 2.0.7 and later).  The file is opened in binary mode and with the\n\
+glibc 2.0.7 and later).  The file is opened in binary mode and with the\n\
 @w{@code{O_EXCL}} flag.\n\
 \n\
 If the optional argument @var{delete} is supplied and is true, the file will\n\
 be deleted automatically when Octave exits.\n\
 \n\
 If successful, @var{fid} is a valid file ID, @var{name} is the name of the\n\
 file, and @var{msg} is an empty string.  Otherwise, @var{fid} is -1,\n\
 @var{name} is empty, and @var{msg} contains a system-dependent error message.\n\
